requires gspell which is not available in Solaris yet
Revisit this for the Gnome 41 update of gedit and potentially pull in
the new dependency

--- a/plugins/meson.build	2021-08-05 13:19:50.438698850 -0700
+++ b/plugins/meson.build	2021-08-05 13:19:56.608626490 -0700
@@ -21,5 +21,4 @@
 subdir('quickopen')
 subdir('snippets')
 subdir('sort')
-subdir('spell')
 subdir('time')
 
--- a/meson.build	2021-08-05 13:23:43.680861735 -0700
+++ b/meson.build	2021-08-05 13:24:38.450865500 -0700
@@ -86,7 +86,6 @@
 libgd_dep = libgd_sp.get_variable('libgd_dep')
 
 libsoup_dep = dependency('libsoup-2.4', version: libsoup_req, required: true)
-gspell_dep = dependency('gspell-1', version: gspell_req, required: true)
 x11_dep = dependency('x11', required: false)
 
 introspection_dep = dependency('gobject-introspection-1.0', version: introspection_req, required: false)
@@ -169,7 +168,6 @@
   'docinfo',
   'modelines',
   'filebrowser',
-  'spell',
   'time',
 ]
 
